tck-runner/src/test/java/org/hibernate/validator/tckrunner/securitymanager/TckRunner.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.tckrunner.securitymanager;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.hibernate.beanvalidation.tck.util.IntegrationTestsMethodSelector;
+import org.testng.ITestResult;
+import org.testng.TestListenerAdapter;
+import org.testng.TestNG;
+import org.testng.xml.XmlMethodSelector;
+import org.testng.xml.XmlPackage;
+import org.testng.xml.XmlSuite;
+import org.testng.xml.XmlTest;
+
+/**
+ * Main class for running the test suite programmatically, for debugging purposes.
+ * <p>
+ * Specify the following arguments when running this test:
+ * <pre>
+ * -Dvalidation.provider=org.hibernate.validator.HibernateValidator
+ * -Djava.security.manager=default
+ * -Djava.security.policy=target/test-classes/test.policy
+ * -Djava.security.debug=access
+ * -DexcludeIntegrationTests=true
+ * -Darquillian.protocol=LocalSecurityManagerTesting
+ * -Dorg.jboss.testharness.spi.StandaloneContainers=org.hibernate.jsr303.tck.util.StandaloneContainersImpl
+ * </pre>
+ *
+ * @author Gunnar Morling
+ */
+public class TckRunner {
```

Comment:
BTW, trying to use this via the Idea, I get:

    access: access allowed ("java.io.FilePermission" "/Applications/Development/IDE/IntelliJ IDEA 14.app/Contents/lib/idea_rt.jar" "read")
    Exception in thread "main" java.lang.ExceptionInInitializerError
    Caused by: java.security.AccessControlException: access denied ("java.util.PropertyPermission" "idea.launcher.bin.path" "read")
	at java.security.AccessControlContext.checkPermission(AccessControlContext.java:457)
	at java.security.AccessController.checkPermission(AccessController.java:884)
	at java.lang.SecurityManager.checkPermission(SecurityManager.java:549)
	at java.lang.SecurityManager.checkPropertyAccess(SecurityManager.java:1294)
	at java.lang.System.getProperty(System.java:714)
	at com.intellij.rt.execution.application.AppMain.<clinit>(AppMain.java:42)